### PR TITLE
Chad switches zones.

### PR DESCRIPTION
**Pull request by b1gd3vd0g**
Apparently Chad's collision with borders never made it to the Blessed Repository. Until now :)

### CODE REVIEW

File: js/classes/entities/midground/chad.js [hunk 1]
```diff
@@ -313,6 +313,11 @@ class Chad {
                             this.pos = new Vector(this.pos.x, entity.boundingBox.bottom);
                         }
                     }
+                    else if (entity instanceof Border) {
+                        LAST_ZONE = ZONE;
```
Comment (calebkrauter) [inline js/classes/entities/midground/chad.js:317]: Love it! Great idea with the Bounding Box as a border!

File: js/scripts/dialog/playground_dialog.js [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+        text:       "Okay dad! GREAT!"
+    },
+    {// index 4
+        speaker:    null,
+        text:       "Use A and D to move right and left!",
+        end: true
+    }
+];
+
+// DialogEater(string[] convo)
```
Comment (calebkrauter) [inline js/scripts/dialog/playground_dialog.js:37]: Looks great! I just added a comment for clarity since I worked on it with you.

**Review by calebkrauter — APPROVED** [review]
Looks great!

**Review by tclaar — APPROVED** [review]
Thanks for adding this!

**Pull request merged**